app/src/main/java/ar/com/wolox/android/training/ui/training/main/tabs/news/NewsPresenter.kt

```diff
@@ -1,17 +1,23 @@
 package ar.com.wolox.android.training.ui.training.main.tabs.news
 
-import android.os.Handler
+import android.annotation.SuppressLint
 import ar.com.wolox.android.training.model.NewsItem
+import ar.com.wolox.android.training.utils.CredentialsSession
 import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import java.text.SimpleDateFormat
+import java.util.Date
 import javax.inject.Inject
 
-private const val ICON = "http://pngimg.com/uploads/android_logo/android_logo_PNG3.png"
```

Comment:
I've never seen this 😮. The kotlin constants should go inside the `companion object` of the class.

Reply:
great!!! i have no idea where i need to declare constants in kotlin ... thanks